packages/public/server/README.md

```diff
@@ -106,3 +106,10 @@ Covers some best practices that new code should comply to.
   There is a possibility to define taxonomy terms in which each edit shall be automatically reviewed.
   This is useful when you want for example set up a sandbox area in which all edits shall be automatically online.
   To archieve this you need to set the configuration variable `autoreview_taxonomy_term_ids` to the list of those taxonomy term ids.
+- **Timestamp for calculating the active community:**
+  There are two routes `/api/user/active-authors` and `/api/user/active-reviewers` for accessing the active authors or active reviewers which are defined by 10 edits / reviews in the last 90 days.
+  However in the local database there are no edits in the last years.
+  Thus when we calculate the active community in the local development environment both events would be empty.
+  So we introduced the configuration `mysql_timestamp_for_active_community`.
+  It is the mysql endpoint of the time frame in which the active community is calculated.
+  In production set it to `CURDATE()`, for development you can something like `Date("2014-01-01")`.
```

Comment:
Actually: can you make this optional so that we don't need to change anything in production?

Reply:
`CURDATE()` is already the default value (see https://github.com/serlo/serlo.org/blob/092ef8ea49a4397a1b9be401422185af50a9ad58/packages/public/server/src/module/User/src/Factory/UserManagerFactory.php#L61 ). I have updated the README.

Reply:
This still needs work, unfortunately. When you remove the value from `definitions.local.php`, PHP  fails with:
```
Fatal error: Uncaught ErrorException: Undefined variable: mysql_timestamp_for_active_community in /usr/local/apache2/htdocs/src/config/autoload/global.php:160 Stack trace: #0 /usr/local/apache2/htdocs/src/config/autoload/global.php(160): exception_error_handler(8, 'Undefined varia...', '/usr/local/apac...', 160, Array) #1 /usr/local/apache2/htdocs/src/vendor/zendframework/zendframework/library/Zend/Config/Factory.php(109): include('/usr/local/apac...') #2 /usr/local/apache2/htdocs/src/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php(376): Zend\Config\Factory::fromFile('config/autoload...') #3 /usr/local/apache2/htdocs/src/vendor/zendframework/zendframework/library/Zend/ModuleManager/Listener/ConfigListener.php(144): Zend\ModuleManager\Listener\ConfigListener->addConfigByPath('config/autoload...', 'glob_path') #4 [internal function]: Zend\ModuleManager\Listener\ConfigListener->onMergeConfig(Object(Zend\ModuleManager\ModuleEvent)) #5 /usr/local/apache2/htdocs/src/vendor/zendframework in /usr/local/apache2/htdocs/src/config/autoload/global.php on line 160
```
which means we need to change `definitions.local.php` in production (breaking change). If you catch that error (e.g. using `isset`), we can continue without any breaking changes.

Reply:
Fixed in https://github.com/serlo/serlo.org/pull/476/commits/6ad6d1b73420251d0ef5574402c32373bc0fdb72